src/main/java/blusunrize/immersiveengineering/common/items/RailgunProjectiles.java

```diff
@@ -129,5 +134,36 @@ public Entity getProjectile(@Nullable Player shooter, ItemStack ammo, Entity def
 				return defaultProjectile;
 			}
 		});
+
+		if(ModList.get().isLoaded("tetra"))
```

Comment:
All of this needs to go into a compat module, see `blusunrize/immersiveengineering/common/util/compat`.